### DISCUSSION TITLE
Require explicit input for "Select Category" dropdown.

### DISCUSSION THREAD

**Discussion by panva** [community/community#3000] [Discussions] [answered]
When in the `/discussions/new` view when one fails to select a category the first one gets automatically assigned. This leads to discussions misplaced in wrong categories. It would be better if the "Select Category" dropdown required a value selected before being able to submit a new discussion.

**Comment by becca** [comment, marked as answer]
We require a selection from the user for a category before the discussion is created. I was not able to repro this issue so hopefully it's no longer a problem 🤞

**Reply by panva** [reply]
It seems this was fixed indeed.
